### DISCUSSION TITLE
wow 4.3.4 npc bot

### DISCUSSION THREAD

**Discussion by xBarselonx** [trickerer/Trinity-Bots#336] [Q&A]
Привет!
В инете нашел ссылку форума по wow 4.3.4 + npc bot. (https://www.mmopro.org/repacks/20159-npcbot-minion-4-3-4-repack.html)
За основу ботов были взяты твои пнц боты. Я не знаю, занимался ли ты ботами для wow 4.3.4., но в любом случае о тебе там упомянули.
Ссылка на разработчика https://bitbucket.org/JCarter4562/. 
Ссылка на патч NPC bot https://bitbucket.org/JCarter4562/tcbotpatches/src/master/
Ссылка на исходные файлы ТС 4.3.4+NPC bot https://bitbucket.org/JCarter4562/tc2botcore/src/434/ (сборку сервера не делал, т.к. последний раз в него вносили изменения 10 лет назад и скорее всего за это время много чего поменялось).

На сегодняшний день я знаю только одних разработчиков которые регулярно занимаются wow 4.3.4 -  https://github.com/The-Cataclysm-Preservation-Project/TrinityCore. Сборку этого сервера делал, о работоспособности ничего сказать не могу, но это другой вопрос.
 
trickerer, возможно ли  сейчас актуализировать вышеуказанные работы по NPC bot и внедрить их в  https://github.com/The-Cataclysm-Preservation-Project/TrinityCore?

**Comment by trickerer** [comment]
> Я не знаю, занимался ли ты ботами для wow 4.3.4., но в любом случае о тебе там упомянули.

Было дело, в истории коммитов можно найти.
Было это давно, в конце концов пришлось дропнуть, так как не было возможности их поддерживать. С тех пор мало что изменилось, никакими другими версиями кроме 3.3.5 я заниматься не планирую. Может когда-нибудь 2.4.3, но это не точно.
